File: clients/rust/src/anonymous_schema13.rs
// AnonymousSchema13 represents a AnonymousSchema13 model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct AnonymousSchema13 {
    #[serde(rename="request")]
    request: Box<crate::AnonymousSchema14>,
    #[serde(rename="changes")]
    changes: Vec<crate::ReservedUnion>,
}

impl AnonymousSchema13 {
    pub fn new(request: crate::AnonymousSchema14, changes: Vec<crate::ReservedUnion>) -> AnonymousSchema13 {
        AnonymousSchema13 {
        Box::new(request),
        changes,
        }
    }
}

File: clients/typescript/src/AnonymousSchema_13.ts
import {AnonymousSchema_14} from './AnonymousSchema_14';
export class AnonymousSchema_13 {
  private _request: AnonymousSchema_14;
  private _changes: (string | any)[];

  constructor(input: {
    request: AnonymousSchema_14,
    changes: (string | any)[],
  }) {
    this._request = input.request;
    this._changes = input.changes;
  }

  get request(): AnonymousSchema_14 { return this._request; }
  set request(request: AnonymousSchema_14) { this._request = request; }

  get changes(): (string | any)[] { return this._changes; }
  set changes(changes: (string | any)[]) { this._changes = changes; }
}
